CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Intermediate bytes of CSI sequences not checked
 - Wayland clipboard integration
 - Use text mouse cursor when mouse mode is temporarily disabled with shift
+- Remove URL styles when selecting text
```

Comment:
I wouldn't know what `URL styles` are and I'd assume I'm not alone here. I think we can describe better what we're doing here.

Not much better, but maybe this would work:

> - Block URL highlight while a selection is active

Open for other suggestions though. I just don't think `Remove URL styles` says a lot.